emsize.py

```diff
@@ -73,9 +88,10 @@ def print_sizes(js_file):
   for line in lines[:-1]:
     print(line)
 
-  print('JS\t\t%s\t0' % js_size)
-  print('Total\t\t%s' % total)
+  print('%s%16d%7d' % ('JS', js_size, 0))
+  print('%s%13s' % ('Total', total))
+  print('\n')
```

Comment:
Why add this?  (BTW, the python print statement add a newline already so this effectively outputs 2 newlines).

Reply:
I know that this produces two new lines.

I did this to mimic the original output of llvm-size.
I must say i only tested it with --format=sysv but there the llvm version also prints out two newlines. If you lnot use option sysv. it looks diffrent.

```
user:~/ProjectLibrary/emscripten_emsize_problem/exampleProject$ llvm-size-13 myapp/myapp.wasm --format=sysv
myapp/myapp.wasm  :
section       size   addr
TYPE           146      0
IMPORT          81      0
FUNCTION        56      0
TABLE            5      0
MEMORY           6      0
GLOBAL          19      0
EXPORT         261      0
ELEM            11      0
CODE         10683      0
DATA           739      0
Total        12007


user:~/ProjectLibrary/emscripten_emsize_problem/exampleProject$
```

Reply:
Sounds good.  Perhaps as a comment here along the lines of "Extra empty line to mimic llmv-size output".   Does gnu size also do this?